routes/chat.js

```diff
@@ -18,6 +18,7 @@ const {
   searchGif,
   listFeaturedGif,
   registerShareGif,
+  initChatFromPost
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `initChatFromPost` from the chat controller is correctly added. This function will be used in the new route handler.
```diff
+   initChatFromPost
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->